### ISSUE TITLE
1D spatial model HW small error in example code

### ISSUE THREAD

**Issue by merrillrudd** [James-Thorson/2016_Spatio-temporal_models#16]
Hi all, Cole and I were tripping up on the 1D spatial model homework - our model wasn't running using the example code from Lab 5. In the covariance matrix, we should be taking the absolute value of the distances (as is shown in the powerpoint), since the data we are reading in are not in order. This helped us. I hope you all realized this before we did, or at least can move past it now! 

- Merrill

**Comment by royburst** [comment]
Yup, had the same issue, then just ordered on location and was set. I just sorted it within the sim function when creating location:

```  y_i = sort(runif( n=n_i, min=32, max=49 )) ```
